Detectors/TRD/calibration/src/DCSProcessor.cxx

```diff
@@ -163,42 +165,135 @@ int DCSProcessor::processDP(const DPCOM& dpcom)
           mLastDPTimeStamps[dpid] = etime;
         }
       }
+
+      if (std::strstr(dpid.get_alias(), "trd_fedEnvTemp") != nullptr) { // DP is trd_fedEnvTemp
+        if (!mFedEnvTempStartTSSet) {
+          mFedEnvTempStartTS = mCurrentTS;
+          mFedEnvTempStartTSSet = true;
+        }
+        auto& dpInfoFedEnvTemp = mTRDDCSFedEnvTemp[dpid];
+        if (etime != mLastDPTimeStamps[dpid]) {
+          // only add data point in case last one was not already read before (every 0.5 hours)
```

Comment:
The objected is processed everytime, but it only updates the CCDB every 0.5h. This is defined in line 208 of `workflow/TRDDCSDataProcessorSpec.h` as there is a time interval for calling `sendDPsoutputFedEnvTemp()` (see [here](https://github.com/leo-barreto/AliceO2/blob/139868a927b12865e0b771294cde0c53a3501a58/Detectors/TRD/calibration/workflow/TRDDCSDataProcessorSpec.h#L207)). Similar to other already defined objects with time intervals for updating. 

The way this is written makes a change to `TRDDCSMinMaxMeanInfo` type object really straightforward, as suggested in the next [conversation](https://github.com/AliceO2Group/AliceO2/pull/12079#discussion_r1362098858l).

The same thing for `trd_cavernHumidity`.

Reply:
ok, but then I would just remove the `(ever 0.5 h)` passage, because if we decide to change it at some point then it will certainly be forgotten to update this comment

Reply:
Done! See commit https://github.com/AliceO2Group/AliceO2/pull/12079/commits/98f02ef5f3e9be889e097f12e7eaa0773111f79d.